action: convert ci can show the metrics collected during conversion

Desiki-high opened the pull request:
Changes:
1. let users specify the file path by output-json, otherwise the json file will be cleared in work directory.
2. We can show the metrics collected during conversion in convert ci, you can see the [preview](https://github.com/Desiki-high/image-service/actions/runs/4988270851#summary-13507820819).

Related PR: https://github.com/dragonflyoss/image-service/pull/1272.
Resolves #1263.

imeoer commented on contrib/nydusify/pkg/converter/converter.go:
```diff
@@ -65,7 +65,11 @@ func Convert(ctx context.Context, opt Opt) error {
-			defer os.RemoveAll(opt.WorkDir)
+			// If the user wants to save the metrics collected during conversion,
+			// we should not clear the work directory.
+			if !opt.OutputJSON {
```
Maybe we should allow users to specify the file path like `nydus-image create --output-json ...`?

Desiki-high replied:
> Maybe we should allow users to specify the file path like `nydus-image create --output-json ...`?

@imeoer  Replace the work directory with a user-specified directory?

Desiki-high replied:
let output-json be a string arg?

Desiki-high replied:
fixed

imeoer commented on contrib/nydusify/cmd/nydusify.go:
```diff
@@ -409,10 +409,10 @@ func main() {
-					Value:   false,
-					Usage:   "Enable saving the metrics collected during conversion in JSON file output.json",
+					Value:   "",
+					Usage:   "File path to save the metrics collected during conversion in JSON format, for example: './metric.json'",
```
'./output.json' should be better, we don't assume that the file only stores metrics for now.

Desiki-high replied:
fxied

Desiki-high committed:
fix: nydusify save metric to specify the file path

We should not save metrics in work directory, otherwise it will be cleared.
We need use to specify the file path, so just change output-json to string opt.

Signed-off-by: Yadong Ding <ding_yadong@foxmail.com>

Desiki-high committed:
action: convert ci show metrics during conversion

Signed-off-by: Yadong Ding <ding_yadong@foxmail.com>

imeoer reviewed (approved):
LGTM!
